framework/include/wifi_manager/wifi_manager.h

```diff
@@ -0,0 +1,170 @@
+	unsigned int ssid_length;				 /**<  Service Set Identification Length  */
+	char passphrase[64];					 /**<  ap passphrase(password)            */
+	unsigned int passphrase_length;			 /**<  ap passphrase length               */
+	wifi_manager_ap_auth_type_e ap_auth_type;	  /**<  @ref wifi_utils_ap_auth_type   */
+	wifi_manager_ap_crypto_type_e ap_crypto_type;  /**<  @ref wifi_utils_ap_crypto_type */
+} wifi_manager_ap_config_s;
+
+/**
+ * @brief Initialize Wi-Fi Manager including starting Wi-Fi interface.
+ * @param[in] callback functions called when wi-fi events happen
+ * @return On success, WIFI_MANAGER_SUCCESS (i.e., 0) is returned. On failure, non-zero value is returned.
+ * @since Tizen RT v1.1
+ */
+wifi_manager_result_e wifi_manager_init(wifi_manager_cb_s *wmcb);
+
+/**
+ * @brief Deinitialize Wi-Fi Manager including stoping Wi-Fi interface.
+ * @param[in] none
+ * @return On success, WIFI_MANAGER_SUCCESS (i.e., 0) is returned. On failure, non-zero value is returned.
+ * @since Tizen RT v1.1
+ */
+wifi_manager_result_e wifi_manager_deinit(void);
+
+/**
+ * @brief Change the Wi-Fi mode to station or AP.
+ * @param[in] Wi-Fi mode (station or AP)
+ * @return On success, WIFI_MANAGER_SUCCESS (i.e., 0) is returned. On failure, non-zero value is returned.
+ * @since Tizen RT v1.1
+ */
+wifi_manager_result_e wifi_manager_set_mode(wifi_manager_mode_e mode);
+
+/**
+ * @brief Retrieve current status of Wi-Fi interface including mode, connection status, ssid, received signal strengh indication, and ip address.
+ * @param[out] retrieved information including  mode, connection s    tatus, ssid, received signal strengh indication, and ip address.
+ * @return On success, WIFI_MANAGER_SUCCESS (i.e., 0) is returned. On failure, non-zero value is returned.
+ * @since Tizen RT v1.1
+ */
+wifi_manager_result_e wifi_manager_get_info(wifi_manager_info_s *info);
+
+/**
+ * @brief Connect to an access point.
+ * @param[in] ssid, passphrase, authentication type, and cryto type of the access point which the wi-fi interface connect to.
+ * @return On success, WIFI_MANAGER_SUCCESS (i.e., 0) is returned. On failure, non-zero value is returned.
+ * @since Tizen RT v1.1
+ */
+wifi_manager_result_e wifi_manager_connect_ap(wifi_manager_ap_config_s *config);
+
+/**
+ * @brief Disconnect from the connected access point
+ * @param[in] none
+ * @return On success, WIFI_MANAGER_SUCCESS (i.e., 0) is returned. On failure, non-zero value is returned.
+ * @since Tizen RT v1.1
+ */
+wifi_manager_result_e wifi_manager_disconnect_ap(void);
+
+/**
+ * @brief Configure ssid, channel, and passphrase of the softAP when the wi-fi interface is set to AP mode.
+ * @param[in] ssid, channel, and passphrase of softAP
+ * @return On success, WIFI_MANAGER_SUCCESS (i.e., 0) is returned. On failure, non-zero value is returned.
+ * @since Tizen RT v1.1
+ */
+wifi_manager_result_e wifi_manager_set_softap_config(wifi_manager_softap_config_s *config);
+
```

Comment:
you opened the "Wi-Fi_Manager ingroup" brace, but not close.
please add below.
/**
 * @}
 */

Reply:
Done

Reply:
you didn't close the }. please check this